package.json

```
@@ -30,10 +30,10 @@
     "ci:publint": "turbo run ci:publint",
     "ci:sherif": "sherif --ignore-package \"./examples/*\" --ignore-package \"./websites/*\" --ignore-package \"./docs/*\" --ignore-dependency \"@tanstack/react-query\" ",
     "ci:test": "turbo run ci:test",
-    "ci:test:watch": "turbo run ci:test:watch --parallel",
     "ci:type": "turbo run ci:type",
     "clean": "turbo run clean",
-    "dev": "turbo run dev --concurrency 20",
+    "dev": "pnpm prepack && concurrently \"pnpm build:watch\" \"pnpm dev:test:watch\"",
```

```suggestion
    "dev": "concurrently \"pnpm build:watch\" \"pnpm dev:test:watch\"",
```
The dependsOn section in turbo.json specifies prepack. I'm wondering if it's additionally necessary to include pnpm prepack in the packages.

```json
   "dev": {
      "dependsOn": ["prepack"],
      "outputs": ["dist/**"],
      "cache": false,
      "persistent": true
    }
```

---

Cool